src/java/picard/vcf/LiftoverVcf.java

```diff
@@ -153,13 +153,17 @@ public static void main(final String[] args) {
             }
             else {
                 // Fix the alleles if we went from positive to negative strand
+                reverseComplementAlleleMap.clear();
                 final List<Allele> alleles = new ArrayList<Allele>();
+
                 for (final Allele oldAllele : ctx.getAlleles()) {
                     if (target.isPositiveStrand() || oldAllele.isSymbolic()) {
                         alleles.add(oldAllele);
                     }
                     else {
```

Comment:
@yfarjoun I don't understand these comments.
1. If it's symbolic we don't try to reverse complement.  That behavior is no different than it was before my change.
2. Why wouldn't this work for indels?  That behavior is also no different than it was before my change.
My change is simply about matching up the Alleles in the Genotypes with the Alleles in the VariantContext.  But I'm not touching the logic of the VariantContext at all.